spec/amp-story-player.md

```diff
@@ -244,6 +244,30 @@ Here's the JSON configuration for opting out of autoplay:
   ...
 ```
 
+## Page Scrolling
+
+By default, the player will scroll the page it's embedded in so that users don't get stuck when scrolling through the player.
```

Comment:
I don't think that's enough context for someone without previous knowledge of the problem to understand why/when they would need to use it.

Let's be explicit:
- Players embedded in a document's content need this enabled so they don't block user scrolling
- Players displayed full screen on a device (eg: lightbox) need this disabled so the page does not scroll in the background